cas/client.c

```diff
@@ -1659,9 +1660,11 @@ M0_INTERNAL int m0_cas_put(struct m0_cas_req      *req,
 		~(COF_CREATE | COF_OVERWRITE | COF_CROW | COF_SYNC_WAIT)) == 0);
 	M0_PRE(m0_cas_id_invariant(index));
 
-	(void)dtx;
 	rc = cas_req_prep(req, index, keys, values, keys->ov_vec.v_nr, flags,
 			  &op);
+	if (rc != 0)
+		return M0_ERR(rc);
+	rc = m0_dtx0_txd_copy(dtx, &op->cg_txd);
```

Comment:
Please make sure if error happens, the "op" is freed in error handling.
It seems not now.

Reply:
Done.